src/features/contacts/form/index.tsx

```diff
@@ -77,12 +73,22 @@ export const ContactForm = ({ contact, updateContact, handleSave, isEdit }: Cont
     }
   };
 
+  const handleSuccess = () => {
```

Comment:
There is almost the same method in the src/components/spreadsheet-import/index.tsx. Is it possible to avoid copy-pasting of that logic?

Reply:
Fixed with common methods